### DISCUSSION TITLE
有關PR的branch name

### DISCUSSION THREAD

**Discussion by soju-sai** [laravel-taiwan/docs#469] [Q&A] [answered]
我看大家開出PR要merge的時候branch name都會變成這種格式 `name:9.x`，但我在local無法用 `git checkout -b name:9.x`，
系統會說

> fatal: 'name:9.x' is not a valid branch name.

請問要怎麼樣讓格式變成那樣，或者是說不用特別寫 `name:9.x` 只要維持 `9.x`，push PR之後就會自動把帳號名稱帶到 `9.x` 之前？例如 `soju-sai:9.x`
謝謝

**Comment by flamerecca** [comment, marked as answer]
不用特別寫 name:9.x

貢獻的方式基本上是
1. fork 到自己的帳號內
2. 調整並 commit 到自己帳號下的 repo
3. 點帳號 repo 的 Contrubute 建立 PR

**Reply by soju-sai** [reply]
感謝！原來是fork，學到了！
